shared/src/main/java/net/blay09/mods/kleeslabs/converter/SlabConverter.java

```diff
@@ -1,14 +1,19 @@
 package net.blay09.mods.kleeslabs.converter;
 
+import net.blay09.mods.balm.api.event.BreakBlockEvent;
 import net.minecraft.world.level.block.state.BlockState;
 import net.minecraft.world.level.block.state.properties.SlabType;
 
 public interface SlabConverter {
 
     BlockState getSingleSlab(BlockState state, SlabType slabType);
 
+    default BlockState getSingleSlab(BreakBlockEvent event, SlabType slabType) {
```

Comment:
I would rather not want to expose Balm's `BreakBlockEvent` as public API (even though SlabConverter itself isn't really public API right now). It would be better if the method just accepted all necessary parameters similar to how Vanilla does it in `Block` methods.

Reply:
I see, okay, let me try to do some changes. Thanks!